.travis.yml

```diff
@@ -54,7 +54,7 @@ matrix:
       env: LIBRABBITMQ_VERSION=v0.5.2 TEST_PHP_ARGS=-m
 
 before_install:
-  - sh provision/install_rabbitmq-c.sh ${LIBRABBITMQ_VERSION}
+  - sudo ./provision/install_rabbitmq-c.sh
```

Comment:
Ughh, commenting on this for the third time, I still can't get used to new GH review features or my comments get lost somehow. 

It looks like `LIBRABBITMQ_VERSION` was removed by accident as it value later retrieved in install script as `$1`.

Also, I'm not entirely sure that sudo is required to run this script. The install script use `sudo` within for commands that need it.

Reply:
I removed the argument to use directly the `LIBRABBITMQ_VERSION` environment variable inside the script.
I can add the argument back if you prefer.

Without the sudo, here is the error when running `cmake`: https://travis-ci.org/swarrot/swarrot/jobs/300596815#L706 (it's on swarrot but it's the same install script)